### PR TITLE
Free Software Foundation Europe #271

### LINKED ISSUE #271: Wild swag opportunity appeared: Free Software Foundation Europe
## Wild swag opportunity appeared!

- [x] I've checked that this isn't a duplicate issue.

Q. Which company offers it?  
A. : Free Software Foundation Europe

Q. Where have you seen the announcement (URL)?  
A. : devRant

Q. What kind of swag (clothing, stickers, ...)?  
A. : Stickers, Booklets, Posters, etc

Q. Have you received swag from this?  
A. : Yes

Q. How long did it take?  
A. : 2 Weeks

Q. Does it ship worldwide?  
A. : Yes

Refer: https://fsfe.org/contribute/spreadtheword.en.html

- [x] I've checked that the promotion is legit.
- [x] I've checked that it's still available to the public.

### PR DESCRIPTION
**Pull request by KartSriv**
Free Software Foundation Europe
Refer: https://fsfe.org/contribute/spreadtheword.en.html

- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

Fixes #271

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -80,6 +80,15 @@
     "dateAdded": "2018-02-18T07:48:58.000Z",
     "tags": ["clothing", "hacktoberfest", "expired", "stickers"]
   },
+  {
+    "name": "FSFE",
+    "difficulty": "easy",
+    "description": "Fill a form and receive promotional materials from Free Software Foundation Europe!",
+    "reference": "https://fsfe.org/contribute/spreadtheword.en.html",
+    "image": "https://i.imgur.com/TuOjguN.jpg",
+    "dateAdded": "2019-05-27T11:42:09.000Z",
```
Comment (aslafy-z) [inline data.json]: The date should be `2019-04-19T04:34:02.000Z` (#271 initial creation date)

File: data.json [hunk 2]
```diff
@@ -80,6 +80,15 @@
     "dateAdded": "2018-02-18T07:48:58.000Z",
     "tags": ["clothing", "hacktoberfest", "expired", "stickers"]
   },
+  {
+    "name": "FSFE",
+    "difficulty": "easy",
+    "description": "Fill a form and receive promotional materials from Free Software Foundation Europe!",
```
Comment (aslafy-z) [inline data.json]: I'd rephrase it like:
```suggestion
    "description": "Fill the form to receive promotional materials from Free Software Foundation Europe!",
```
WDYT?

**Comment by aslafy-z** [comment]
Thank you @KartSriv! Please review the comments I left and I'd be able to approve your change :+1:

File: data.json [hunk 3]
```diff
@@ -80,6 +80,15 @@
     "dateAdded": "2018-02-18T07:48:58.000Z",
     "tags": ["clothing", "hacktoberfest", "expired", "stickers"]
   },
+  {
+    "name": "FSFE",
+    "difficulty": "easy",
+    "description": "Fill the form to receive promotional materials from Free Software Foundation Europe!",
+    "reference": "https://fsfe.org/contribute/spreadtheword.en.html",
+    "image": "https://i.imgur.com/TuOjguN.jpg",
```
Comment (aslafy-z) [inline data.json]:
```suggestion
    "image": "https://i.ibb.co/P9XVfPm/FSFE-stickers-medium.jpg",
```



I'd go for this image instead, it's clearer so we can really know what's inside!

**Comment by vikaspotluri123** [comment]
Looks good - do we want to specify that this opportunity is more geared towards larger groups?

**Comment by KartSriv** [comment]
> Looks good - do we want to specify that this opportunity is more geared towards larger groups?

I think we can skip that cause I don't see any implications not mentioning that.

**Comment by aslafy-z** [comment]
Thank you @KartSriv! It's live now!

**Comment by KartSriv** [comment]
Your welcome!
